Makefile

```diff
@@ -302,7 +302,7 @@ dockerpull:
 .PHONY: test
 test: $(VENV) config-templates $(TRANSLATION_FILES) $(DOC_BUILD)
 	export $(shell cat $(ENV_FILE)) && ${PYTHON} ./scripts/pg_ready.py
-	PYTHONPATH=${PYTHONPATH} S3_TESTS=$(S3_TESTS) ${NOSE} --verbosity=2 --cover-erase  tests/ -e "(.*e2e.*|test_search|test_sphinxapi)"
+	PYTHONPATH=${PYTHONPATH} S3_TESTS=$(S3_TESTS) ${NOSE} --verbosity=2 --cover-erase  tests/ -e "(.*e2e.*|test_sphinxapi)"
```

Comment:
`test_sphinxapi` should also be removed. It is part of the `service-search-wsgi` now.